contracts/HATVaults.sol

```diff
@@ -400,6 +406,7 @@ contract  HATVaults is Governable, ReentrancyGuard {
         pool.rewardPerShare = pool.rewardPerShare + (reward * 1e12 / totalShares);
         pool.lastRewardBlock = block.number;
         pool.lastProcessedTotalAllocPoint = lastPoolUpdate;
+        emit UpdatePool(_pid, pool.rewardPerShare);
```

Comment:
so one reason against this event here is that updatepool is called on each deposit and withdrawal, which already have events..

Reply:
i was thinking maybe we want to know what happened (the new reward per share)